.claude/skills/storybook-testing/SKILL.md

```
@@ -0,0 +1,854 @@
+
+   // ✅ Correct (with fn())
+   args: {
+     onSubmit: fn(async () => ({ success: true }));
+   }
+   ```
+
+## Integration with Project
+
+**File Structure:**
+
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)